izug/refegovservice/profiles/default/types/izug.refegovservice.egovservice.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    name="izug.refegovservice.egovservice"
+    meta_type="Dexterity FTI"
+    i18n:domain="izug.refegovservice">
+
+  <!-- Basic properties -->
+  <property
+      i18n:translate=""
+      name="title">EGov Service</property>
+  <property
+      i18n:translate=""
+      name="description">Description of an Government Service</property>
+
+  <property name="allow_discussion">False</property>
+  <property name="factory">izug.refegovservice.egovservice</property>
+  <property name="icon_expr">string:${portal_url}/document_icon.png</property>
+  <property name="link_target"></property>
+
+  <!-- Hierarchy control -->
+  <property name="global_allow">True</property>
+  <!-- Schema, class and security -->
+  <property name="add_permission">izug.refegovservice.AddEGovService</property>
+  <property name="klass">izug.refegovservice.content.e_gov_service.EGovService</property>
+  <property name="schema">izug.refegovservice.content.e_gov_service.IEGovService</property>
+
+  <!-- Enabled behaviors -->
+  <property name="behaviors" purge="false">
+
+    <element value="plone.app.content.interfaces.INameFromTitle"/>
+    <element value="plone.app.dexterity.behaviors.metadata.IBasic"/>
+    <!-- categorization includes the language field -->
+    <element value="plone.app.dexterity.behaviors.metadata.ICategorization"/>
+    <!-- TODO - are some of the below unnecessary? -->
+    <element value="plone.app.dexterity.behaviors.metadata.IPublication"/>
+    <element value="plone.app.dexterity.behaviors.metadata.IOwnership"/>
+    <element value="plone.app.dexterity.behaviors.exclfromnav.IExcludeFromNavigation"/>
+
+    <!-- <element value="plone.app.referenceablebehavior.referenceable.IReferenceable" /> -->
+    <!--<element value="plone.translatable" />-->
+  </property>
+
+  <!-- View information -->
+  <property name="add_view_expr">string:${folder_url}/++add++izug.refegovservice.egovservice</property>
+  <property name="default_view">egovleistung_view</property>
+  <property name="default_view_fallback">False</property>
+  <property name="immediate_view">view</property>
+  <property name="view_methods">
+    <element value="view"/>
```

Comment:
Is there a reason that the default view is `egovleistung_view` and the immediate view is just `view` and both are selectable in the display menu? Is there a need for two views?
I'd prefer to only have one.

Reply:
This originates from the archetypes defn of `default/types/EgovLeistung.xml` - I can't see any reason why, so will change it